pkg/volume/glusterfs/glusterfs.go

```diff
@@ -67,13 +67,14 @@ var _ volume.Deleter = &glusterfsVolumeDeleter{}
 const (
 	glusterfsPluginName            = "kubernetes.io/glusterfs"
 	volPrefix                      = "vol_"
-	dynamicEpSvcPrefix             = "glusterfs-dynamic-"
+	dynamicEpSvcPrefix             = "glusterfs-dynamic"
 	replicaCount                   = 3
 	durabilityType                 = "replicate"
 	secretKeyName                  = "key" // key name used in secret
 	gciLinuxGlusterMountBinaryPath = "/sbin/mount.glusterfs"
 	defaultGidMin                  = 2000
 	defaultGidMax                  = math.MaxInt32
+	maxCustomEpNamePrefixLen       = 26
```

Comment:
Please add a comment where this number comes from (max length of name - length of uuid?)

Reply:
Done.